src/CalcViewModel/DataLoaders/CurrencyDataLoader.cpp

```diff
@@ -502,7 +502,22 @@ bool CurrencyDataLoader::TryParseStaticData(_In_ String ^ rawJson, _Inout_ vecto
     staticData.resize(size_t{ data->Size });
     for (unsigned int i = 0; i < data->Size; i++)
     {
-        JsonObject ^ obj = data->GetAt(i)->GetObject();
+        JsonObject ^ obj;
+        try
+        {
+            obj = data->GetAt(i)->GetObject();
+        }
+        catch (COMException ^ e)
```

Comment:
Given that we're not modifying anything from it...
```suggestion
        catch (const COMException& e)
```

Reply:
As we've discussed, let's ignore this suggestion for now.